OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -35,6 +35,7 @@ public class ViewportControllerWidget : Widget
 		public int EdgeCornerScrollThreshold = 35;
 
 		int2? joystickScrollStart, joystickScrollEnd;
+		bool isStandardScrolling;
```

Comment:
It would be good if we could suppress the order cursor when this is true.  This should fix

> Order cursor still appears but doesn't execute orders til RMB is released to exit scrolling mode.